colour/colorimetry/spectrum.py

```diff
@@ -839,7 +835,7 @@ def items(self):
             Spectral power distribution data generator.
         """
 
-        return self.__iter__()
+        return np.array(list(self.__iter__()))
```

Comment:
Turns out that I didn't manage to get `np.fromiter` to interpret correctly the input:

`return np.fromiter(self.__iter__(), dtype=np.float_)`

```
  File "/Users/kelsolaar/Documents/Development/colour-science/colour/colour/colorimetry/spectrum.py", line 838, in items
    return np.fromiter(self.__iter__(), dtype=np.float_)
ValueError: setting an array element with a sequence.
```

Seems like `np.fromiter` can only construct 1D arrays: http://stackoverflow.com/a/34020332/931625